README.rst

```diff
@@ -35,11 +35,15 @@ If installing alongside another CWL implementation then::
 
   pip install cwltool
 
-To install from source::
+To install from source
 
-  git clone https://github.com/common-workflow-language/cwltool.git
-  cd cwltool && python setup.py install
-  cd cwlref-runner && python setup.py install  # co-installing? skip this
+#. Install virtualenv via pip: ``pip install virtualenv``
+#. Clone the cwltool: ``git clone https://github.com/common-workflow-language/cwltool.git``
+#. Switch to cwltool directory: ``cd cwltool``
+#. Create a virtual environment: ``virtualenv -p python2 venv``, use ``python3`` if you want to use in Python 3 runtime.
+#. To begin using the virtual environment, it needs to be activated: ``source venv/bin/activate``
+#. Install cwltool in the virtual environment: ``pip install .``
+#. Check the version and make sure cwltool is invoked correctly: ``cwltool --version``
```

Comment:
Great! Can you reformat this so it is easier to copy and paste into a terminal all at once instead of line by line?